### ISSUE TITLE
'Outstream' object has no attribute closed

### ISSUE THREAD

**Issue by jacobzweig** [tqdm/tqdm#126]
Using Canopy and Python 2.7.10, upon running 
`from tqdm import tqdm`

I receive the following error

```
AttributeError                            Traceback (most recent call last)
<ipython-input-7-6b39a9324e6c> in <module>()
----> 1 from tqdm import tqdm

c:\users\j\appdata\local\enthought\canopy\user\src\tqdm\tqdm\__init__.py in <module>()
----> 1 from ._tqdm import tqdm
      2 from ._tqdm import trange
      3 from ._tqdm_gui import tqdm_gui
      4 from ._tqdm_gui import tgrange
      5 from ._tqdm_pandas import tqdm_pandas

c:\users\j\appdata\local\enthought\canopy\user\src\tqdm\tqdm\_tqdm.py in <module>()
     12 from __future__ import division, absolute_import
     13 # import compatibility functions and utilities
---> 14 from ._utils import _supports_unicode, _environ_cols_wrapper, _range, _unich,      15     _term_move_up, _unicode, WeakSet
     16 import sys

c:\users\j\appdata\local\enthought\canopy\user\src\tqdm\tqdm\_utils.py in <module>()
     19 try:  # pragma: no cover
     20     import colorama
---> 21     colorama.init()
     22 except ImportError:  # pragma: no cover
     23     colorama = None

C:\Users\j\AppData\Local\Enthought\Canopy\User\lib\site-packages\colorama\initialise.pyc in init(autoreset, convert, strip, wrap)
     35     else:
     36         sys.stdout = wrapped_stdout = ---> 37             wrap_stream(orig_stdout, convert, strip, autoreset, wrap)
     38     if sys.stderr is None:
     39         wrapped_stderr = None

C:\Users\j\AppData\Local\Enthought\Canopy\User\lib\site-packages\colorama\initialise.pyc in wrap_stream(stream, convert, strip, autoreset, wrap)
     74     if wrap:
     75         wrapper = AnsiToWin32(stream,
---> 76             convert=convert, strip=strip, autoreset=autoreset)
     77         if wrapper.should_wrap():
     78             stream = wrapper.stream

C:\Users\j\AppData\Local\Enthought\Canopy\User\lib\site-packages\colorama\ansitowin32.pyc in __init__(self, wrapped, convert, strip, autoreset)
     65         # should we strip ANSI sequences from our output?
     66         if strip is None:
---> 67             strip = conversion_supported or (not wrapped.closed and not is_a_tty(wrapped))
     68         self.strip = strip
     69 

AttributeError: 'OutStream' object has no attribute 'closed'
```

**Comment by lrq3000** [comment]
Arg, another issue from colorama, but I guess that can't be avoided since
it's a so difficult project to maintain.

I guess that you are using Python inside the Canopy GUI and not inside a
standard Windows cmd prompt, am I correct?

2016-02-11 17:53 GMT+01:00 Jacob <notifications@github.com>:

**Comment by jacobzweig** [comment]
Yep, only happens when running in the gui, I don't receive the error from
Python within the cmd prompt.

**Comment by jacobzweig** [comment]
Noticed a fix for this issue in [this commit of colorama](https://github.com/tartley/colorama/commit/1244a00ae1accfeca3b4b1a85a3b718a920dd6bd)

Referenced from: https://github.com/tartley/colorama/issues/85

**Comment by casperdcl** [comment]
I think this was the same issue I had which was (partially) fixed in #125

**Comment by jacobzweig** [comment]
Thanks @casperdcl and @lrq3000  - the issue was fixed for me by updating colorama to the version referenced in my comment above.

**Comment by lrq3000** [comment]
Ok thank's for reporting Jacob, wow the colorama guys are quite fast,
that's the second time they fix an issue just after it gets reported here
and before we have the time to report it to them XD

2016-02-11 19:41 GMT+01:00 Jacob <notifications@github.com>:

**Comment by casperdcl** [comment]
Great I reverted the temp colorama fix in #125... `bpython` still doesn't work, while `bpython-curses` works but only ascii. Not tqdm's fault (`print unichr(257)` fails) but still annoying.
